build.gradle.kts

```diff
@@ -19,7 +19,7 @@ dependencies {
 fun log4j(
     vararg modules: String
 ): List<String> = modules.map { module ->
-    "org.apache.logging.log4j:log4j-$module:2.10.0"
+    "org.apache.logging.log4j:log4j-$module:2.17.0"
```

Comment:
Dep locks need an update too, right?

Reply:
There's no dependency locking in this module.

Reply:
Roger that :+1: